docs/GlobalAPI.md

```diff
@@ -878,3 +878,23 @@ const add = (a, b) => a + b;
 
 test.todo('add should be associative');
 ```
+
+### `test.failing(name, fn)`
+
+Also under the aliases: `it.failing(name, fn)`, `xtest.failing(name, fn)`, `xit.failing(name, fn)`, `it.skip.failing(name, fn)`, `it.only.failing(name, fn)`
```

Comment:
Perhaps in alphabetical order: `test.concurent`, `test.each`, `test.failing`, ... , `test.todo`.

Also `skip` and `only` most probably are not aliases. `test.failing` and `test.skip.failing` is not the same thing.

Reply:
Applied suggestion :)